libs/ui/src/components/Explorer/_queries/useExplorerData.ts

```diff
@@ -40,6 +50,87 @@ const _mapping = (data: ExplorerQuery, libraryId: string, availableLangs: string
     };
 };
 
+const getQueryFilters = (filters: IExplorerFilter[]): RecordFilterInput[] =>
+    filters
+        .filter(
+            ({value, condition}) =>
+                value !== null ||
+                [RecordFilterCondition.IS_EMPTY, RecordFilterCondition.IS_NOT_EMPTY].includes(condition)
+        )
+        .reduce((acc, filter, index) => {
+            if (index !== 0) {
+                acc.push({
+                    operator: RecordFilterOperator.AND
+                });
+            }
+            if (filter.attribute.format === AttributeFormat.date) {
+                acc = acc.concat(_getDateRequestFilters(filter));
+            } else {
+                acc.push({
+                    field: filter.field,
+                    condition: filter.condition,
+                    value: filter.value
+                });
+            }
```

Comment:
**todo:** C'est un `array.map` ça ? ça pourrait être fait avant le `push` qui a été fait autrement dans une autre PR.

Reply:
non c'est pas un array map, ligne 67 on cancatene 2 tableaux

Reply:
un `array.flatmap` alors

Reply:
je n'aime pas le mélange entre la manipulation des données et l'ajout de l'opérateur `AND`

Reply:
J'ai fait un rebase et j'ai utilisé ce qu'à fait Teddy., du coup plus de mélange